try-it.md

```diff
@@ -24,6 +25,14 @@ permalink: /try-it.html
 <hr>
 ## Instructions
 
+### Note on naming
```

Comment:
You could use here an admonition by doing:

> note "Naming"
>For the v1alpha3 release, the Cluster API provider for metal3 was renamed from
> Cluster API provider BareMetal to Cluster API provider Metal3. Hence, if working
> with v1alpha1 or v1alpha2, it will be Cluster API provider Baremetal (CAPBM) in
> this documentation and deployments, but from v1alpha3 onwards it will be
> Cluster API provider Metal3 (CAPM3).

Reply:
done, thank you

Reply:
![imagen](https://user-images.githubusercontent.com/312463/75540565-8e3a7000-5a1c-11ea-8c28-3b03e62bb9e7.png)

Cool!!